src/components/CreateAnalysis/CreateAnalysisDetail.tsx

```diff
@@ -10,34 +9,35 @@ import SelectionStudy from "./SelectionStudy";
 import FileLookup from './FileLookup';
 import Error from "../../shared/error";
 import { PageSection, PageSectionVariants } from "@patternfly/react-core";
+import { DcmImage } from "../../context/reducers/dicomImagesReducer";
+import Utils from "../../shared/utils";
 
 interface CreateAnalysisDetailProps {
   setIsExpanded: Dispatch<SetStateAction<boolean>>,
   submitAnalysis: () => void
 }
 
-const CreateAnalysisDetail: React.FC<CreateAnalysisDetailProps> = ({setIsExpanded, submitAnalysis}) => {
-  const { state: { dcmImages, createAnalysis }, dispatch } = useContext(AppContext);
-  const { selectedStudyUIDs } = createAnalysis;
+const CreateAnalysisDetail: React.FC<CreateAnalysisDetailProps> = ({ setIsExpanded, submitAnalysis }) => {
+  const { state: { createAnalysis, dcmImages } } = useContext(AppContext);
   const [isXray, setIsXray] = useState(false);
+  const [patientName, setPatientName] = useState('');
+  const [patientBirthdate, setPatientBirthdate] = useState('');
+  const [patientSex, setPatientSex] = useState('');
```

Comment:
Please, use double quotemarks over single quotemarks.
```suggestion
  const [patientName, setPatientName] = useState("");
  const [patientBirthdate, setPatientBirthdate] = useState("");
  const [patientSex, setPatientSex] = useState("");
```